src/vs/workbench/services/editor/browser/editorService.ts

```diff
@@ -869,6 +869,8 @@ export class DelegatingEditorService implements IEditorService {
 
 	createInput(input: IResourceEditor): IEditorInput { return this.editorService.createInput(input); }
 
+	findTargetGroup(input: IEditorInput, options?: IEditorOptions, group?: OpenInEditorGroup): IEditorGroup { return this.editorService.findTargetGroup(input, options, group); }
```

Comment:
@bpasero Can you please take a look at how this change exposes `findTargetGroup`. Is there a better way to accomplish this?